app/controllers/application_controller.rb

```diff
@@ -25,6 +25,8 @@ def page_param
   def set_locale
     supported_languages = %w[en de es pt-BR fr]
 
+    supported_languages.each { |l| I18n.load_path += Dir[Rails.root.join('config', 'locales', l, '*.yml').to_s] }
```

Comment:
:warning: This is evaluated at each request.  I guess that `I18n.load_path` modification invalidates all cached data.  Maybe this could be moved into `config/initializers` which is only evaluated one at startup.

Reply:
Thanks for the tip, i'll take a look tonight.